src/components/MovieForm/MovieForm.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import styles from './MovieForm.module.css';
+
+export const MovieForm = ({ movieInfo = {}, onSubmit }) => {
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    const data = Object.fromEntries(formData);
+    onSubmit(data); // todo fix
+  };
+
+  return (
+    <form id="movie_form" className={styles.form} onSubmit={handleSubmit}>
+      <div className={styles.div}>
+        <div className={styles.left}>
+          <label htmlFor="title">title</label>
+          <input
+            id="title"
```

Comment:
```suggestion
            id="title"
            name="title"
```
And the same for others `input` fields.